### PR TITLE
 english name change and Partial sinicization #159

### PR DESCRIPTION
**Pull request by David0817**
* change the account to profile
 * change the wallet to account
 * account page for Chinese

### CODE REVIEW

**Comment by evias** [comment]
Missing rename of `Create a new account` to `Create a new profile`. Please make sure to also update the **import / create account** modules. Thank you.

Screenshot 2020-04-01 at 6 26 09 PM

File: src/services/WalletService.ts [hunk 1]
```diff
@@ -281,7 +281,7 @@ export class WalletService extends AbstractService {
 
     return new WalletsModel(new Map<string, any>([
       [ 'accountName', currentAccount.getIdentifier() ],
-      [ 'name', 'Seed Wallet 1' ],
+      [ 'name', 'Seed Account 1' ],
```
Comment (evias) [inline src/services/WalletService.ts:284]: this one uses space

File: src/views/pages/accounts/import-account/wallet-selection/WalletSelectionTs.ts [hunk 2]
```diff
@@ -256,7 +256,7 @@ export default class WalletSelectionTs extends Vue {
     return simpleWallets.map((simpleWallet, i) =>
       new WalletsModel(new Map<string, any>([
         [ 'accountName', this.currentAccount.values.get('accountName') ],
-        [ 'name', `Seed Wallet${indexes[i] + 1}` ],
+        [ 'name', `Seed Account${indexes[i] + 1}` ],
```
Comment (evias) [inline src/views/pages/accounts/import-account/wallet-selection/WalletSelectionTs.ts:259]: this one is missing the space.

Comment (David0817) [reply src/views/pages/accounts/import-account/wallet-selection/WalletSelectionTs.ts]: fixed

**Pull request merged**